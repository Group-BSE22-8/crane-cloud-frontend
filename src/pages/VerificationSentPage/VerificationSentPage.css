.VerificationPageContainer {
  background: #008ac1;
  color: #ffffff;
  display: flex;
  flex-direction: column;
  flex: 1;
}

.VerificationPageHeader {
  flex: 1;
}

.VerificationPageMain {
  flex: 1;
  display: grid;
  grid-template-columns: 1fr;
  justify-items: center;
  align-items: center;
}

.VerificationPageContent {
  padding: 2rem;
}

.VerificationPageContent h2 {
  color: #f7b21f;
}

.VerificationPageContent p span {
  color: #f7b21f;
}

.ResendLinkForm {
  display: flex;
  flex-direction: column;
}

.ResendLinkForm .ResendFormInputs {
  display: grid;
  grid-template-columns: 1fr;
  gap: 1rem;
}

.ResendLinkForm .ResendFormInputs .LoginErrorDiv {
  width: 25rem;
}
