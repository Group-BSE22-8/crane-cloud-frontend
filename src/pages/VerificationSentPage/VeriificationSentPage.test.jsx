/* eslint-disable no-undef */
import React from "react";
import { shallow } from "enzyme";
import VerificationSentPage, { mapStateToProps } from "./";

const VerificationSentPageProps = {
  data: [],
  match: { params: { token: "my_token" } },
};

describe("Testing the App Metrics Page component", () => {
  const WrapperVerificationSentPage = VerificationSentPage.WrappedComponent;
  const VerificationSentPageComponent = shallow(
    <WrapperVerificationSentPage {...VerificationSentPageProps} />
  );
  it("should match the snapshot for VerificationSentPage after adding props", () => {
    VerificationSentPageComponent.setProps(VerificationSentPageProps);
    expect(VerificationSentPageComponent).toBeDefined();
  });
  it("matchs the VerificationSentPage component snapshot", () => {
    expect(VerificationSentPageComponent).toMatchSnapshot();
  });
});

describe("Testing the exported mapstate to props and dispatch", () => {
  it("match the Project settings page mapstostate", () => {
    expect(
      mapStateToProps({
        user: { data: [] },
      })
    ).toEqual({
      data: [],
    });
  });
});
