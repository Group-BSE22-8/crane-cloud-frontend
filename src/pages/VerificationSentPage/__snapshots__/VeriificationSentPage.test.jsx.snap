// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the App Metrics Page component matchs the VerificationSentPage component snapshot 1`] = `ShallowWrapper {}`;
